frontend/src/lib/components/PropertyGroupFilters/PropertyGroupFilters.tsx

```diff
@@ -0,0 +1,180 @@
+                                            onChange={(properties) => {
+                                                setPropertyFilters(properties, propertyGroupIndex)
+                                            }}
+                                            pageKey={`trends-filters-${propertyGroupIndex}`}
+                                            taxonomicGroupTypes={taxonomicGroupTypes}
+                                            eventNames={eventNames}
+                                            propertyGroupType={group.type}
+                                        />
+                                    </div>
+                                    {propertyGroupIndex !== filtersWithNew?.groups.length - 1 && (
+                                        <Row className="text-small primary-alt">
+                                            <b>{filtersWithNew.type}</b>
+                                        </Row>
+                                    )}
+                                </>
+                            )
+                        })}
+                    </BindLogic>
+                </div>
+            )}
+            <div>
+                <Button
+                    style={{
+                        color: 'var(--primary)',
+                        border: 'none',
+                        boxShadow: 'none',
+                    }}
+                    className="mb"
+                    icon={<PlusOutlined />}
+                    onClick={() => addFilterGroup()}
+                >
+                    Add filter group
+                </Button>
+            </div>
+        </>
+    )
+}
+
+export enum AndOr {
+    AND = 'AND',
+    OR = 'OR',
+}
+
+interface AndOrFilterSelectProps {
+    onChange: (type: AndOr) => void
+    value: string
+    topLevelFilter?: boolean
+}
+
+export function AndOrFilterSelect({ onChange, value, topLevelFilter }: AndOrFilterSelectProps): JSX.Element {
+    return (
+        <Row align="middle" className="and-or-filter">
+            <span className="ml-05">Match</span>
+            <Select
+                optionLabelProp="label"
+                dropdownClassName="and-or-filter-select"
+                style={{ marginLeft: 8, marginRight: 8 }}
+                defaultValue={AndOr.AND}
+                onChange={(type) => onChange(type)}
+                dropdownMatchSelectWidth={false}
+                placement={topLevelFilter ? 'bottomRight' : 'bottomLeft'}
+            >
+                <Select.Option value={AndOr.AND} label="all" className="condition-option">
+                    <Row>
+                        <div className={`condition-text ${value === AndOr.AND ? 'selected' : ''}`}>{AndOr.AND}</div>
+                        <Col>
+                            <div>
+                                <b>All filter</b>{' '}
+                            </div>
+                            <div>All filters must be met (logical and)</div>
```

Comment:
This " (logical and/or)" detail seems really redundant here – someone who knows what a "logical and/or/xor/etc." is will easily understand that from the big "AND"/"OR" label on the left already combined with the description.

Reply:
@clarkus